opm/parser/eclipse/Deck/DeckKeyword.hpp

```diff
@@ -39,7 +39,7 @@ namespace Opm {
 
         const std::string& name() const;
         void setFixedSize();
-        void setLocation(const std::string& fileName, int lineNumber);
+        void setLocation(const std::pair<const std::string&, std::size_t>& location);
```

Comment:
Strictly speaking, you need to `#include <utility>` if you're going to declare a function that takes a `std::pair`.

Reply:
OK